### ISSUE TITLE
Making a 2016 gridpack

### ISSUE THREAD

**Issue by DryRun** [cms-sw/genproductions#2542]
I was wondering how the logic for 2016 PDFs is supposed to work in the gridpack creation scripts. Particular these lines, https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/Utilities/gridpack_helpers.sh#L12-L23, which control the default PDFs used for 2016 vs. 2017. Should this line:

`elif grep -q -e "\$DEFAULT_PDF_SETS" -e "\$DEFAULT_PDF_MEMBERS" $CARDSDIR/${name}_run_card.dat; then`

be something like this instead:

`elif grep -q -e "\$DEFAULT_2016_PDF_SETS" -e "\$DEFAULT_PDF_MEMBERS" $CARDSDIR/${name}_run_card.dat; then` ? It seems like the first "elif" block will never be entered. 

Thanks,
David

**Comment by agrohsje** [comment]
Hi @DryRun . I hope @kdlong or @qliphy can comment more on the logic. I must admit it is not clear to me how the current splitting is supposed to work.

**Comment by qliphy** [comment]
Currently for 2017/2018, PDF can be handled automatically, if one set in run_card.dat the following as in GEN recommendation:
                   $DEFAULT_PDF_SETS = lhaid                                  [1]
                   $DEFAULT_PDF_MEMBERS = reweight_PDF           [2]

However, for 2016, one has to set it explicitly by hand as one may want a different central PDF.  
In this case, as @DryRun mentioned, as there is no [1] [2] set as above, 
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/Utilities/gridpack_helpers.sh#L18-L23
this will not leave the run_card.dat as default.
On the other hand, due to the following:
https://github.com/cms-sw/genproductions/blob/master/Utilities/scripts/getMG5_aMC_PDFInputs.py#L18
https://github.com/cms-sw/genproductions/blob/master/Utilities/python/PDFSetsChooserTools.py#L10-L16
the reweight PDF sets will just follow 2017.
In runcms

**Comment by kdlong** [comment]
Right. This was on my todo list for so long it fell off. If someone would like to have it, it's quite straightforward to implement, I can provide input at least.

**Comment by agrohsje** [comment]
@DryRun Would you be available to do it with the help of @kdlong ?

**Comment by DryRun** [comment]
Hi @agrohsje - sure, I'd be happy to.
